src/framework/base.model.ts

```diff
@@ -175,4 +178,53 @@ export default abstract class BaseModel {
         return this.schema.bulkWrite(upserts);
 
     }
+    getCategories() {
+        return this.schema.aggregate(
```

Comment:
take the array to field "pipeline" and use to enter, for example:
const pipeline = [
   {
      $match: {
          _id:ObjectId()                        
      }
   },
   {
      $project:{
         _id:0
      }
   }
]